packages/babel-plugin/src/utils/css-builders.tsx

```diff
@@ -281,12 +402,21 @@ const extractTemplateLiteral = (node: t.TemplateLiteral, meta: Metadata): CSSOut
       return acc + q.value.raw + interpolation.value;
     }
 
-    if (t.isObjectExpression(interpolation) || isCompiledCSSTemplateLiteral(interpolation, meta)) {
+    if (
+      t.isObjectExpression(interpolation) ||
+      isCompiledCSSTemplateLiteral(interpolation, meta) ||
+      (nodeExpression &&
+        t.isArrowFunctionExpression(nodeExpression) &&
+        t.isConditionalExpression(nodeExpression.body))
+    ) {
       // We found something that looks like CSS.
       const result = buildCss(interpolation, updatedMeta);
-      variables.push(...result.variables);
-      css.push(...result.css);
-      return acc;
+
+      if (result.css.length > 0) {
```

Comment:
Do we need to check this? What's the edge case here?

Reply:
@MonicaOlejniczak yes, we need it. In lines 408-411 I introduced an extra condition to deal with conditional mixins. Without result.css.length > 0, we have a variety of tests failing